scripts/update-submodule.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
+
+# Copyright 2021 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Update python-base submodule and collect release notes.
+# Usage:
+#   scripts/update-submodule.sh
+#
+# Create a commit "generated python-base update" and send a PR to the main repo.
+# NOTE: not all python-base changes are user-facing, and you may want to remove
+# some of the non-user-facing release notes from CHANGELOG.md.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+repo_root="$(git rev-parse --show-toplevel)"
+declare -r repo_root
+cd "${repo_root}"
+
+source scripts/util/changelog.sh
+
+# TODO(roycaihw): make the script send a PR by default (standalone mode), and
+# have an option to make the script reusable by other release automations.
+TARGET_RELEASE=${TARGET_RELEASE:-"v$(grep "^CLIENT_VERSION = \"" scripts/constants.py | sed "s/CLIENT_VERSION = \"//g" | sed "s/\"//g")"}
```

Comment:
so TARGET_RELEASE is an env var that can be set when running the script, better put it in the Usage: section above to make it clear.

Reply:
done